python-rhsm/src/rhsm/connection.py

```diff
@@ -1405,6 +1405,20 @@ def getSubscriptionList(self, owner_key):
         results = self.conn.request_get(method)
         return results
 
+    def updateSubscriptionList(self, owner_key, auto_create_owner=False, lazy_regen=True):
```

Comment:
Hm, I've chosen this approach, so it's consitant with other methods like `regenEntitlementCertificates` or `getEntitlementList`, etc. I've used Cadnlepin defaults as values here, but it make sense to leave it None as well. I'll change it